pkg/kn/commands/source/ping/update.go

```diff
@@ -19,10 +19,14 @@ import (
 	"fmt"
 
 	"github.com/spf13/cobra"
+	"k8s.io/client-go/util/retry"
+	"knative.dev/client/pkg/kn/commands/service"
```

Comment:
Why do you need this dependency ? Normally commands should stay independent and don't refer to each other to avoid cyclic dependency and allow them easily to be refactored/moved. 

If there is some code that can be reused, I'd propose to move it to a shared package and use it from there in service and here.

Reply:
This is because I imported the MaxUpdateRetries from service package. I will change this and create a variable in the same directory.

Reply:
Thanks !